llvmlite/tests/test_dylib.py

```diff
@@ -0,0 +1,35 @@
+from . import TestCase
```

Comment:
This doesn't deserve to be in its separate test file. You can put in test_binding.py.